awx/main/management/commands/cleanup_host_metrics.py

```diff
@@ -1,22 +1,22 @@
-from awx.main.models import HostMetric
 from django.core.management.base import BaseCommand
 from django.conf import settings
+from awx.main.tasks.host_metrics import HostMetricTask
 
 
 class Command(BaseCommand):
     """
-    Run soft-deleting of HostMetrics
+    This command provides cleanup task for HostMetric model.
+    There are two modes, which run in following order:
+    - soft cleanup
+    - - Perform soft-deletion of all host metrics last automated 12 months ago or before.
+        This is the same as issuing a DELETE request to /api/v2/host_metrics/N/ for all host metrics that match the criteria.
+    - - updates columns delete, deleted_counter and last_deleted
+    - hard cleanup
+    - - Permanently erase from the database all host metrics last automated 36 months ago or before.
+        This operation happens after the soft deletion has finished.
     """
```

Comment:
What if you tried assigning this docstring to a `help` variable as in the Django docs?

https://docs.djangoproject.com/en/4.2/howto/custom-management-commands/

Reply:
@AlanCoding we discussed that previously. The `help` variable is here below docstring and should be one-liner. 
It's not supposed to be long description like the docstring (also according to the doc you posted ^)